model/Software/Vocabularies/SoftwarePurpose.md

```diff
@@ -27,6 +27,7 @@ about the context in which the Element exists.
 - container: the Element is a container image which can be used by a container runtime application
 - data: Element is data
 - device: the Element refers to a chipset, processor, or electronic board
+- deviceDriver: the referred element represents software that controls hardware devices
```

Comment:
Can we keep standardized with the others?   s/the referred element/the Element/ ?